micropip/externals/mousebender/simple.py

```diff
@@ -7,13 +7,12 @@
 import warnings
 from typing import Any, Dict, List, Optional, Union, Literal, TypeAlias, TypedDict
 
-import packaging.utils
+import micropip._vendored.packaging.utils as packaging_utils
```

Comment:
(No need to address in this PR)

both `_vendored` and `externals` directory vendors external packages, so I think we can merge them into a single directory

Reply:
Thanks, I think `_vendored` is a better name. At the beginning, the `external` directory contained subset of `pip`, and it had a `_vendor` directory in it ([ref](https://github.com/pyodide/pyodide/tree/0.18.0/packages/micropip/src/micropip/externals)).